app/src/main/java/kr/ac/kaist/team888/mylittlefont/FontCanvasView.java

```diff
@@ -89,7 +96,7 @@ public void onDraw(Canvas canvas) {
     locator.invalidate(canvasRegion);
 
     if (skeletonView) {
-      for (Path path : locator.getPaths()) {
+      for (Path path : locator.getSkeletonPaths()) {
         canvas.drawPath(path, skeletonPaint);
       }
 
```

Comment:
Can you make contour also drawn in skeletonView with stroke but not fill? Viewing contour with skeleton seems more clear.